content/en/tracing/trace_explorer/_index.md

```diff
@@ -127,13 +127,17 @@ All spans indexed by custom retention filters *and* the intelligent retention fi
 {{% /tab %}}
 {{% tab "Timeseries View" %}}
 
-All spans indexed by custom retention filters (*not* the intelligent retention filter) are available to be searched when using trace analytics.
+All spans indexed by custom retention filters **and** the intelligent retention filter are available to be searched when using trace analytics.
```

Comment:
```suggestion
All spans indexed by custom retention filters or the intelligent retention filter are available to be searched when using trace analytics.
```

OR means "any span indexed by either the custom retention filter or the intelligent retention filter is available"
AND means "only spans that have been indexed by both the custom retention filter and intelligent retention filter are available"